pegasus/router.rb

```diff
@@ -406,13 +406,24 @@ def post_process_html_from_markdown(full_document)
       full_document
     end
 
+    def log_drupal_link(dir, uri)
+      if dir == 'drupal.code.org'
+        Honeybadger.notify(
+          error_class: "Link to v3.sites/drupal.code.org",
+          error_message: "#{uri} fell through to the base config directory"
+        )
```

Comment:
it looks like you could add something like `environment_name: drupal_#{Rack.env}` to the options here?

http://www.rubydoc.info/github/honeybadger-io/honeybadger-ruby/Honeybadger.notify